java/clients/src/test/java/sleeper/clients/admin/TableNamesReportTest.java

```diff
@@ -55,4 +57,33 @@ void shouldPrintTableNamesReportWhenChosen() throws Exception {
         order.verify(in.mock).promptLine(any());
         order.verifyNoMoreInteractions();
     }
+
+    @Test
+    void shouldPrintOnlineAndOfflineTableNames() throws Exception {
+        // Given
+        setInstanceTables(createValidInstanceProperties(),
+                List.of("test-table-1", "test-table-2"),
+                List.of("offline-table-1", "offline-table-2"));
```

Comment:
There's no indication here that the second list is offline tables. You could just have a separate takeTablesOffline method which would call TakeTableOffline against the table index.

It might be simpler to create the tables as one operation and then take them offline, rather than mixing the two things.